project/ScalafixBuild.scala

```diff
@@ -247,15 +238,22 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
         "olafurpg@gmail.com",
         url("https://geirsson.com")
       )
-    )
-  )
-
-  private val PreviousScalaVersion = Map(
-    "2.11.12" -> "2.11.11",
-    "2.12.4" -> "2.12.3"
-  )
-
-  override def projectSettings: Seq[Def.Setting[_]] = List(
+    ),
+    publishTo := {
```

Comment:
What happens if you move these to `buildSettings`?

Reply:
What is the difference?

Reply:
reload performance, `projectSettings` are evaluated for every project while `buildSettings` are only evaluated once for the build. scalafix has 64 projects, and adding 2.13/native support would increase that number even more.

Reply:
Done.